IRremoteESP8266.h

```diff
@@ -232,18 +232,19 @@ class IRsend
   void sendKelvinator(unsigned char data[]);
   void sendSherwood(unsigned long data, int nbits=32, unsigned int repeat=1);
   void sendMitsubishiAC(unsigned char data[]);
-  void enableIROut(int khz);
+  void enableIROut(unsigned int khz, uint8_t duty=50);
```

Comment:
You are right. This should serve in both cases. I am not an expert in C++ either and I never happened to overload in this way. :)